wiki/Article_Styling_Criteria/en.md

```diff
@@ -349,65 +513,206 @@ All beatmapset difficulty URLs looks like this: `https://osu.ppy.sh/beatmapsets/
+
+Adding the full country names in the alt text is optional.
 
 ### Tables
 
-Tables use the GFM syntax.
+Tables in the osu! wiki support headings along the first row. If you cannot create a table because something would not work without HTML (e.g. lists inside tables), you need to reconsider the layout you are trying to use.
 
-If you cannot create a table because _something_ would not work without HTML (e.g. lists inside tables), you need to reconsider the layout you are trying to use.
+Tables must never be beautified. They must have a vertical bar (`|`) on the left and right sides and the tect of each cell must be padded with one space on both sides. Empty cells must use a vertical bar (`|`) followed by two spaces then another vertical bar (`|`).
 
-Here is a tool to help visualize tables:
+The delimiter row (the row that separates the table head from the table body) must use only three characters per column (and be padded with a space on both sides), which must look like one of the following:
 
-- [Markdown Tables Generator](http://www.tablesgenerator.com/markdown_tables)
+- `---` (for left align)
+- `:--` (for left align)
+- `:-:` (for centre align)
+- `--:` (for right align)
 
-#### Beautifying tables
+The left align delimiter must be done consistently throughout the entire article.
 
-It is up to you whether or not if you want to beautify the tables.
+---
+
+For example:
 
-They will appear nicer when editing them; however, for those who use text wrapping, those tables will appear as a clumped mess. Another thing to note, a slight change in a beautified table will require you to fix the spacing of every cell, depending on how big of change you made.
+```md
+| Team "Picturesque" Red | Score | Team "Statuesque" Blue | Average Beatmap Stars |
+| :-- | :-: | --: | :-- |
+| **peppy** | 5 - 2 | pippi | 9.3 stars |
+| Aiko | 1 - 6 | **Alisa** | 4.2 stars |
+| Ryūta | 3 - 4 | **Yuzu** | 5.1 stars |
+| **Taikonator** | 7 - 0 | Tama | 13.37 stars |
+| Maria | No Contest | Mocha |  |
+```
 
-Some tools for beautifying tables are:
+### Blockquotes
 
-- VS Code's Native Beautifier
-  - this will require you to use [VS Code](https://code.visualstudio.com/)
-- [Markdown Table Formatter](http://markdowntable.com/)
-  - alignment syntax (`:`) will not parse correctly
+Usage of blockquotes is limited to quoting text from someone. It must not be used to format text otherwise.
 
 ### Horizontal bar
 
 Use horizontal bars sparingly. A few uses of the horizontal bar may include (but is not limited to):
 
 - separating images from text
-- marking the end of a "section" within the same section
+- marking the end of a part of text within the same section
 
-When using the horizontal bar, make sure there is an empty line above and below the markup. Otherwise, osu!web will think the text above is a heading level 2.
+When using the horizontal bar, make sure there is an empty line above and below the markup. Otherwise, it will be parsed as a level 2 heading.
```

Comment:
Add the amount of hyphens(`-`) for consistency.

Reply:
should or must?

Reply:
Should, since it’s just a code thing.